README.md

```diff
@@ -88,3 +97,61 @@ cd ..
 ```
 Ran git commit along the way and finally pushed to this repo.
 
+## To enable a Hermes event
+
+Android
+
+In your build.gradle file make sure to have the 
```

Comment:
is all of this already in the build.gradle file? If so, we don't have to repeat it all in the readme here.

Update - I checked, I see it in your /android/app/build.gradle.

My request - can you write "see /android/app/build.gradle" here under "To enable a Hermes event"?

Then in android/app/build.gradle can you put a comment somewhere like
```
// This is for enabling Hermes

```
I'm curious about the "androidx.swiperefreshlayout:swiperefreshlayout:1.0.0" and "facebook.flipper" things that I see, if those are for Hermes then put "// This is for Hermes" as it's not clear what swiperefresh and flipper are for.

If those are for some UI behaviors (non-Hermes) then put a comment clarifying what they're for.

Reply:
Do you have a piece of documentation/tutorial/blog you worked off of for this? If you still can find it, if you could paste the link into the PR that'd be great. It's helpful if we're ever debugging Hermes stuff in the future.

Reply:
Done